Moto/G8Plus/res/values/config.xml

```diff
@@ -0,0 +1,211 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <bool name="config_dozeAfterScreenOff">false</bool>
+    <bool name="config_dynamic_bind_ims">true</bool>
```

Comment:
[ims] not really useful since it's in VoLTE-CAF which is automatically enabled when IMS apk is installed but well